lib/portage/dbapi/vartree.py

```diff
@@ -2750,7 +2750,14 @@ def _unmerge_pkgfiles(self, pkgfiles, others_in_slot):
             # process symlinks second-to-last, directories last.
             mydirs = set()
 
-            uninstall_ignore = shlex.split(self.settings.get("UNINSTALL_IGNORE", ""))
+            uninstall_ignore = self.settings.get("UNINSTALL_IGNORE", "")
+            if "packdebug" in self.settings.features:
+                # For packdebug, we don't want to clean up old debuginfo
+                # tarballs that we made, as we want to keep serving them
+                # to binhost clients for some time.
+                uninstall_ignore += " /usr/lib/debug/.tarball"
```

Comment:
This doesn't seem to be working correctly for me now, though I thought I'd tested it, so not sure if I broke it:
```
$ emerge -v1 app-misc/hello
[...]
--- /usr/lib/
--- /usr/lib/debug/
--- /usr/lib/debug/.tarball/
--- /usr/lib/debug/.tarball/app-misc/
--- /usr/lib/debug/.tarball/app-misc/hello/
>>> /usr/lib/debug/.tarball/app-misc/hello/hello-2.12.2-3-debug.tar.xz
[...]
--- replaced dir /usr/share
--- replaced obj /usr/lib/debug/usr/bin/hello.debug
--- replaced dir /usr/lib/debug/usr/bin
--- replaced dir /usr/lib/debug/usr
<<<          obj /usr/lib/debug/.tarball/app-misc/hello/hello-2.12.2-2-debug.tar.xz
```

The tarball gets removed unexpectedly.

Reply:
Yes, that fixes it.

Reply:
Hm, does that imply that `PKG_INSTALL_MASK=/usr/lib/debug/*` won't stop packing up `.tarball`?